README.md

```diff
@@ -79,4 +79,60 @@ Your permissions will be the same between both namespaces, and you will be able
 2. `make run` - Runs Kots Admin
 3. In a new terminal, navigate to the kots project.
 4. `okteto exec bash` - Runs bash interactively in the kots pod.
-5. `./bin/kost {{COMMAND}}` - Run the kots commands you need. 
\ No newline at end of file
+5. `./bin/kost {{COMMAND}}` - Run the kots commands you need.
+
+### Build V2 (EXPERIMENTAL)
+
+#### Description
+
+This new iteration of our Okteto workspace has significant changes and requires a new workflow by developers.
+
+#### Why
+
+We've been trying to optimize our build times and make developing on Okteto as frictionless as possible.  However, we've realized that there are some fundemental issues with our current strategy, such as:
+
+1. Builds take place in two places (buildkit, in dev containers).  This causes issues with cache sharing, image size, etc.
+2. Spike in resources for development containers.  Some of our apps put a heavy strain on resources when built, this require us to either give them a lot of resources while in development mode (which can be long-lasting) or starve them of resources and bottleneck builds.
+3. Unable to quickly/easily deploy kubernetes manifest changes.
+
+#### Solution
+
+This V2 work flow attempts to solve these issues by:
+
+1. Build application only on the buildkit servers so that the cache lives in one place and image sizes stay lean.  This excluded applications that have live reloading (web).
+2. Only use development containers where needed.  (web apps, schema hero, etc)
+3. Update the Okteto manifest to the new schema which allows for separating build and deploy specs, allowing us to run `okteto deploy` and only deploy the manifest. 
+
+#### Reference
+
+| Blah | Syntax | Description                                                                                                |
+| ---- | ------ |------------------------------------------------------------------------------------------------------------|
+| Build and Deploy | `okteto pipeline deploy -f okteto-v2.yml` | Runs both build and deploy sections of the Okteto manifest. Perfect for updating or creating a namespace.  |
+| Build single service | `okteto build -f okteto-v2.yml {{SERVICE_NAME}}` | Builds the named service (kotsadm, kotsadm-web, kotsadm-migrations) and pushes it to the Okteto registry. |
+| Deploy | `okteto deploy -f okteto-v2.yml` | Deploys the kubernetes manifests. If there were builds before this command, the new images will be used in the deployment. | 
+| Development mode | `okteto up -f okteto-v2.yml` | Prompts the use for what container to put into development mode.  Only web and migrations should appear at this time. |
+
+#### Warning
+
+Because this new workflow is experimental, we still have the old workflow in the project.  If you are using the new workflow, and fail to provide the `-f` flag with the v2 manifest, you will be invoking the old workflow.
+
+#### Example workflow: kotsadm change
+
+1. `okteto pipeline deploy -f okteto-v2.yml`
```

Comment:
i understand this is experimental but once ga could some of these be aliased and combined via make commands?

Reply:
IMO, it's ok that the user knows it's okteto under the hood.  If we wrap these, it will work 99% of the time, but it might be a lot more context loading to understand how to fix when it breaks.  I like the approach documented here where you learn the `okteto` CLI commands